cinema/models.py

```diff
@@ -1,3 +1,5 @@
+from typing import Type
```

Comment:
The import `Type` from `typing` is not used in the code. Consider removing it to clean up the imports.